release/internal/config/config.go

```diff
@@ -44,6 +44,9 @@ type Config struct {
 	// GitRemote is the remote for the git repository
 	GitRemote string `envconfig:"GIT_REMOTE" default:"origin"`
 
+	// GitRepo is the repository name
+	GitRepo string `envconfig:"GIT_REPO_NAME" default:"calico"`
```

Comment:
I think this should be a CLI argument

Reply:
As usage of this value do eventually require the Organization (which isn't a CLI arg), I will prefer to keep this 
This is similar to what we have with operator presently.

Reply:
I think they should just both be CLI args? We can use the env var support of the CLI tool framework if this is needed in CI (but I don't think it is?)

CLI arguments are self-documenting, so using those where possible should be a priority.